DOCS/man/input.rst

```diff
@@ -1935,13 +1935,13 @@ information. They can be manipulated with the ``set``/``add``/``cycle``
 commands, and retrieved with ``show-text``, or anything else that uses property
 expansion. (See `Property Expansion`_.)
 
-The property name is annotated with RW to indicate whether the property is
-generally writable.
-
 If an option is referenced, the property will normally take/return exactly the
 same values as the option. In these cases, properties are merely a way to change
 an option at runtime.
 
+Note that some properties are unavailable until playback starts. See `Details on
```

Comment:
> Note that some properties are unavailable until playback starts.

I'm not sure I like this wording. It's oddly specific. Some properties won't be available until "something" happens. Example would be that `video-params` won't be available after audio file playback starts. Playback start doesn't guarantee that all properties are available.

Scripts are not suppose to pull the data and especially not at init time. But init time is not special, because we might go to idle state after playback, and many properties won't be available. 

We need to educate users that they should observe properties or expect that any and all of them may or may not be present depending on the current player state.

I think it is the core of scripting in mpv and such vague one-liner won't help really.

Reply:
Did you see the rest of the commit? I mentioned observe_property in details on script initialization.